packages/plugin-core/src/views/LocalGraphView.ts

```diff
@@ -0,0 +1,178 @@
+import {
+  DendronTreeViewKey,
+  DMessage,
+  DMessageEnum,
+  GraphEvents,
+  GraphViewMessage,
+  GraphViewMessageEnum,
+  NoteProps,
+  NoteUtils,
+  OnDidChangeActiveTextEditorMsg,
+} from "@dendronhq/common-all";
+import { MetadataService, WorkspaceUtils } from "@dendronhq/engine-server";
+import _ from "lodash";
+import * as vscode from "vscode";
+import { GotoNoteCommand } from "../commands/GotoNote";
+import { IDendronExtension } from "../dendronExtensionInterface";
+import { Logger } from "../logger";
+import { GraphStyleService } from "../styles";
+import { AnalyticsUtils } from "../utils/analytics";
+import { VSCodeUtils } from "../vsCodeUtils";
+import { WebViewUtils } from "./utils";
+
+export class LocalGraphView implements vscode.WebviewViewProvider {
+  public static readonly viewType = DendronTreeViewKey.LOCAL_GRAPH_VIEW;
+  private _view?: vscode.WebviewView;
+  private _ext: IDendronExtension;
+  constructor(extension: IDendronExtension) {
+    this._ext = extension;
+    this._ext.context.subscriptions.push(
+      vscode.window.onDidChangeActiveTextEditor(this.onOpenTextDocument, this)
+    );
+  }
+
+  public postMessage(msg: DMessage) {
+    this._view?.webview.postMessage(msg);
+  }
+
+  public async resolveWebviewView(
+    webviewView: vscode.WebviewView,
+    _context: vscode.WebviewViewResolveContext<unknown>,
+    _token: vscode.CancellationToken
+  ) {
+    this._view = webviewView;
+    await WebViewUtils.prepareTreeView({
+      ext: this._ext,
+      key: DendronTreeViewKey.LOCAL_GRAPH_VIEW,
+      webviewView,
+    });
+    webviewView.webview.onDidReceiveMessage(
+      this.onDidReceiveMessageHandler,
+      this
+    );
+  }
+  async onDidReceiveMessageHandler(msg: GraphViewMessage) {
+    const ctx = "LocalGraphView:onDidReceiveMessage";
+    Logger.info({ ctx, data: msg });
+    switch (msg.type) {
+      case GraphViewMessageEnum.onSelect: {
+        const note = this._ext.getEngine().notes[msg.data.id];
+        await new GotoNoteCommand(this._ext).execute({
+          qs: note.fname,
+          vault: note.vault,
+        });
+        AnalyticsUtils.track(GraphEvents.LocalGraphSidePanel, {
+          message: GraphViewMessageEnum.onSelect,
```

Comment:
for consistency, let's try to use the same property names across different instances of this `GraphEvents.LocalGraphSidePanel` event type.  So maybe something like 

```typescript
AnalyticsUtils.track(GraphEvents.GraphPanelUsed, {
          type: "NodeClicked",
}
```